Sources/Sentry/SentryClient.m

```diff
@@ -190,6 +191,15 @@ - (SentryEvent *_Nullable)prepareEvent:(SentryEvent *)event withScope:(SentrySco
         event.environment = environment;
     }
 
+    NSMutableDictionary *sdk =
+        @{ @"name" : SentryMeta.sdkName, @"version" : SentryMeta.versionString }.mutableCopy;
+    if (nil != sdk && nil == event.sdk) {
+        if (event.extra[@"__sentry_sdk_integrations"]) {
+            [sdk setValue:event.extra[@"__sentry_sdk_integrations"] forKey:@"integrations"];
+        }
+        event.sdk = sdk;
+    }
+
```

Comment:
It would be nice to add a test for this, but `SentryClient` is not testable right now. We already have this on our list. This is just an internal note to me to change this soon.